PublicMapping/district-builder-2
# Create and list projects

## Overview
Create and list projects.

### Demo
![district_builder_create_project](https://user-images.githubusercontent.com/2926237/80095895-b20ac580-8536-11ea-8315-f02a13efd677.gif)

## Notes
I added listing projects to this PR to make it demoable without realizing there was a separate issue for it (#70 )

This PR includes a couple of very small unrelated changes (0317b2e and be25bed)

## Testing Instructions
As per the demo above:
* Ensure you have at least one region config (there's no UI for this yet but there are examples of how to do this in https://github.com/PublicMapping/district-builder-2/pull/87)
* Go to the main page and click 'Create project'
* Fill out the form and should be able to create a project successfully
* Should be redirected to home page and see your new project listed

Then log in as a different user and make sure don't see any projects.

Closes #69 
Closes #70
